Remotion/Data/DomainObjects.UnitTests/Linq/IntegrationTests/ObjectIDIntegrationTest.cs

```diff
@@ -202,10 +202,10 @@ public void Query_UsesClassID_OnReferencedValue ()
     [Test]
     public void QueryWithContainsInWhere_OnCollection_WithObjectIDValues ()
     {
-      var possibleItems = new[] { DomainObjectIDs.Order1.Value, DomainObjectIDs.Order3.Value };
+      var possibleItems = new[] { (Guid)DomainObjectIDs.Order1.Value, (Guid)DomainObjectIDs.Order3.Value };
       var orders =
           from o in QueryFactory.CreateLinqQuery<Order>()
-          where possibleItems.Contains(o.ID.Value)
+          where possibleItems.Contains((Guid)o.ID.Value)
           select o;
```

Comment:
Since you have changed the test spec in the integration test, I think we instead need to copy the test and do both versions, once with object and once with Guid